recipes/vectorbt/meta.yaml

```
@@ -50,7 +50,7 @@ test:
     - pytest-dotenv
   commands:
     - pip check
-    - pytest tests
+    - pytest tests || true
```

Is this meant to skip tests?

---

How about adding comments based on the ones of fbb080a?

```suggestion
    # Five tests fail.
    - pytest tests || true
```

---

I think I was awaiting upstream feedback to finalize. This works for me

---

Thank you, Claire, I presume ?